### PR TITLE
Use Python3

### PR DESCRIPTION
**Pull request by jgawor**
Python2 is EOL in 2020.

### Checklist:

- [X] Read the [Code of Conduct](https://github.com/appsody/website/blob/master/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md).

- [X] Followed the [commit message guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).

- [ ] Stack adheres to [Appsody stack structure](https://github.com/appsody/website/blob/master/content/docs/stacks/stack-structure.md).

### Modifying an existing stack:

- [ ] Updated the stack version in `stack.yaml`

### Contributing a new stack:

- Describe how application dependencies are managed:

- Explain how Appsody file watcher is utilized:

- Describe other Appsody environment variables defined by the stack image:

- Describe any limitations and known issues:

### CODE REVIEW

File: ci/create_codewind_index.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 for file in os.listdir(assets_dir):
     if fnmatch.fnmatch(file, '*index.yaml'):
-        with open(assets_dir + file, 'r') as yamlFile, open(assets_dir + os.path.splitext(file)[0] + ".json", 'w') as jsonFile:
+        with open(assets_dir + file, 'r') as yamlFile, open(assets_dir + os.path.splitext(file)[0] + ".json", 'wb') as jsonFile:
```
Comment (skoh7645) [inline ci/create_codewind_index.py]: Why did you change the "w" to "wb" here? This will always be writing a `.json` file which is a text file

Comment (jgawor) [reply ci/create_codewind_index.py]: Without it, you'll get `TypeError: write() argument must be str, not bytes` error. There might be other ways around it but it's related to a change between Python2 and 3 and how it handles strings.

Comment (skoh7645) [reply ci/create_codewind_index.py]: Yes, I understand this now, thanks
